include/librealsense2/hpp/rs_device.hpp

```diff
@@ -1,5 +1,5 @@
 // License: Apache 2.0. See LICENSE file in root directory.
-// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
```

Comment:
In general (for all files) this should keep the original year: 2017-2024 or 2017-24.